llvm/test/Analysis/CtxProfAnalysis/flatten-and-annotate.ll

```diff
@@ -0,0 +1,88 @@
+; REQUIRES: x86_64-linux
```

Comment:
Can the test case be augmented to show counter merging (from different contexts) after flattening?

Reply:
ptal